lib/core/src/server/manager/manager-config.js

```diff
@@ -51,29 +51,34 @@ async function getManagerWebpackConfig(options, presets) {
   const babelOptions = await presets.apply('babel', {}, { ...options, typescriptOptions });
 
   const autoRefs = await getAutoRefs(options);
-  const refs = await presets.apply('refs', undefined, options);
+  const definedRefs = await presets.apply('refs', undefined, options);
   const entries = await presets.apply('managerEntries', [], options);
 
-  if (refs) {
+  const refs = {};
+
+  if (autoRefs && autoRefs.length) {
     autoRefs.forEach(({ id, url, title }) => {
       refs[id] = {
         id,
-        url,
+        url: url.replace(/\/$/, ''),
```

Comment:
@ndelangen refactor this into a helper function like `sanitizeUrl`, comment why it's needed